src/liboslexec/pointcloud.cpp

```diff
@@ -41,8 +41,7 @@ namespace { // anon
 
 #if USE_PARTIO
 
-class PointCloud {
-public:
+struct PointCloud {
```

Comment:
Not something I'm going to lose sleep over either. I was mainly just wondering why you did it.